igor-monitor-travis/src/main/java/com/netflix/spinnaker/igor/travis/service/TravisService.java

```diff
@@ -338,6 +346,18 @@ public Build getBuild(String repoSlug, int buildNumber) {
         .collect(Collectors.toList());
   }
 
+  private List<V3Build> getBuildsForSpecificRepos(Collection<String> repositories) {
+    return repositories.stream()
```

Comment:
Have you tested using a parallellStream here? Don't know how the Travis API will react, but seems like a simple way to speed this up

Reply:
Good idea! I've updated with parallelStream and it looks like it's working quite nicely!

```
2020-06-23 14:04:30.796  INFO 35991 --- [   scheduling-1] c.n.s.igor.travis.service.TravisService  : fetching builds for repo: adobe/ops-cli
2020-06-23 14:04:30.796  INFO 35991 --- [nPool-worker-19] c.n.s.igor.travis.service.TravisService  : fetching builds for repo: adobe/alloy
```